### PR TITLE
[ID] Add "TAG"

### PR DESCRIPTION
**Pull request by mousesalt**
Add Indonesian translation of [TAG](https://osu.ppy.sh/wiki/en/Beatmap/TAG).

### CODE REVIEW

File: wiki/Beatmap/TAG/id.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+tags:
+  - multiplayer
+  - co-op
+---
+
+# TAG
+
+**TAG** adalah istilah yang digunakan untuk mendeskripsikan mode tim [multiplayer](/wiki/Multi) yang dinamakan [Tag Coop atau Tag Team Vs](/wiki/Multi#tag-coop-/-tag-team-vs), di mana pemain-pemain mencoba untuk menyelesaikan sebuah [beatmap](/wiki/id/Beatmap) secara kooperatif. Jenis beatmap yang secara khusus didesain untuk mode osu!, disebut sebagai **Beatmap TAG/tingkat kesulitan**.
+
+Dalam mode ini, setiap pemain bertanggung jawab untuk satu [komboset](/wiki/id/Beatmapping/Combo) dengan beralih di antara komboset sepanjang lagu. Alasan ini diambil karena, beatmap TAG dirancang sedemikian rupa sehingga kombosetnya ditempatkan lebih jauh dan lebih kompleks dari biasanya, sehingga hampir tidak mungkin beatmap dimainkan oleh satu pemain.
```
Comment (wowcake) [inline wiki/Beatmap/TAG/id.md:12]:
```suggestion
Dalam mode ini, setiap pemain bertanggung jawab untuk satu [komboset](/wiki/id/Beatmapping/Combo) dengan beralih di antara komboset sepanjang lagu. Alasan ini diambil karena, beatmap TAG dirancang sedemikian rupa sehingga kombosetnya ditempatkan lebih jauh dan lebih kompleks dari biasanya, sehingga beatmap tersebut hampir tidak mungkin bisa dimainkan oleh hanya satu pemain.

File: wiki/Beatmap/TAG/id.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+[Tingkat kesulitan](/wiki/id/Beatmap/Difficulty) TAG biasanya dilambangkan dengan penggunaan istilah "TAG" dalam nama kesulitan beatmap. Pada nama kesulitan "TAG" biasanya diikuti dengan angka, dengan khusus menunjukkan jumlah pemain yang harus bergabung untuk beatmap tersebut (misalnya, "TAG2" untuk dua pemain atau "TAG4" untuk empat pemain).
+
+Karena sifatnya, tingkat kesulitan TAG tidak diperingkatkan atau Loved, karena sengaja dirancang agar hampir mustahil dimainkan oleh satu pemain.
```
Comment (wowcake) [inline wiki/Beatmap/TAG/id.md:16]:
```suggestion
Karena sifatnya, tingkat kesulitan TAG tidak diranked atau Loved, karena tingkat kesulitan TAG disengaja untuk dirancang agar hampir mustahil dimainkan oleh hanya satu pemain.

**Review by wowcake — CHANGES_REQUESTED** [review]

**Comment by wowcake** [comment]
checked

File: wiki/Beatmap/TAG/id.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - multiplayer
+  - co-op
+---
+
+# TAG
+
+**TAG** adalah istilah yang digunakan untuk mendeskripsikan mode tim [multiplayer](/wiki/Multi) yang dinamakan [Tag Coop atau Tag Team Vs](/wiki/Multi#tag-coop-/-tag-team-vs), di mana pemain-pemain mencoba untuk menyelesaikan sebuah [beatmap](/wiki/id/Beatmap) secara kooperatif. Jenis beatmap yang secara khusus didesain untuk mode osu!, disebut sebagai **Beatmap TAG/tingkat kesulitan**.
```
Comment (ZeroPyrozen) [inline wiki/Beatmap/TAG/id.md:10]:
```suggestion
**TAG** adalah istilah yang digunakan untuk mendeskripsikan mode tim [multiplayer](/wiki/Multi) yang dinamakan [Tag Coop atau Tag Team Vs](/wiki/Multi#tag-coop-/-tag-team-vs), di mana pemain-pemain mencoba untuk menyelesaikan sebuah [beatmap](/wiki/id/Beatmap) secara kooperatif. Jenis beatmap yang secara khusus didesain untuk mode osu!, disebut sebagai **beatmap/tingkat kesulitan TAG**.
```
**TAG** diletakkan pada bagian terakhir agar **tingkat kesulitan** posisinya sama dengan **beatmap**

File: wiki/Beatmap/TAG/id.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+Dalam mode ini, setiap pemain bertanggung jawab untuk satu [komboset](/wiki/id/Beatmapping/Combo) dengan beralih di antara komboset sepanjang lagu. Alasan ini diambil karena, beatmap TAG dirancang sedemikian rupa sehingga kombosetnya ditempatkan lebih jauh dan lebih kompleks dari biasanya, sehingga beatmap tersebut hampir tidak mungkin bisa dimainkan oleh hanya satu pemain.
+
+[Tingkat kesulitan](/wiki/id/Beatmap/Difficulty) TAG biasanya dilambangkan dengan penggunaan istilah "TAG" dalam nama kesulitan beatmap. Pada nama kesulitan "TAG" biasanya diikuti dengan angka, dengan khusus menunjukkan jumlah pemain yang harus bergabung untuk beatmap tersebut (misalnya, "TAG2" untuk dua pemain atau "TAG4" untuk empat pemain).
+
+Karena sifatnya, tingkat kesulitan TAG tidak diranked atau Loved, karena tingkat kesulitan TAG disengaja dirancang agar hampir mustahil dimainkan oleh hanya satu pemain.
```
Comment (ZeroPyrozen) [inline wiki/Beatmap/TAG/id.md:16]:
```suggestion
Karena sifatnya, tingkat kesulitan TAG memiliki status beatmap Unranked atau Loved, karena tingkat kesulitan TAG sengaja dirancang agar hampir mustahil dimainkan oleh satu pemain.
```

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]
Mohon dilihat kembali masukkan saya.

File: wiki/Beatmap/TAG/id.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+tags:
+  - multiplayer
+  - co-op
+---
+
+# TAG
+
+**TAG** adalah istilah yang digunakan untuk mendeskripsikan mode tim [multiplayer](/wiki/Multi) yang dinamakan [Tag Coop atau Tag Team Vs](/wiki/Multi#tag-coop-/-tag-team-vs), di mana pemain-pemain mencoba untuk menyelesaikan sebuah [beatmap](/wiki/id/Beatmap) secara kooperatif. Jenis beatmap yang secara khusus didesain untuk mode osu!, disebut sebagai **Beatmap TAG/tingkat kesulitan**.
+
+Dalam mode ini, setiap pemain bertanggung jawab untuk satu [komboset](/wiki/id/Beatmapping/Combo) dengan beralih di antara komboset sepanjang lagu. Alasan ini diambil karena, beatmap TAG dirancang sedemikian rupa sehingga kombosetnya ditempatkan lebih jauh dan lebih kompleks dari biasanya, sehingga beatmap tersebut hampir tidak mungkin bisa dimainkan oleh hanya satu pemain.
```
Comment (wowcake) [inline wiki/Beatmap/TAG/id.md:12]:
```suggestion
Dalam mode ini, setiap pemain bertanggung jawab untuk satu [komboset](/wiki/id/Beatmapping/Combo) dengan pemain bergantian di antara komboset sepanjang lagu. Alasan ini diambil karena, beatmap TAG dirancang sedemikian rupa sehingga kombosetnya ditempatkan lebih jauh dan lebih kompleks dari biasanya, sehingga beatmap tersebut hampir tidak mungkin bisa dimainkan oleh hanya satu pemain.

**Review by wowcake — CHANGES_REQUESTED** [review]
will recheck again later

File: wiki/Beatmap/TAG/id.md (hunk 4 above)
Comment (mousesalt) [reply wiki/Beatmap/TAG/id.md:16]: `tidak diranked` sound like something common usage. changed to another way instead

File: wiki/Beatmap/TAG/id.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+Dalam mode ini, setiap pemain bertanggung jawab untuk satu [komboset](/wiki/id/Beatmapping/Combo) dengan pemain bergantian di antara komboset sepanjang lagu. Alasan ini diambil karena, beatmap TAG dirancang sedemikian rupa sehingga kombosetnya ditempatkan lebih jauh dan lebih kompleks dari biasanya, sehingga beatmap tersebut hampir tidak mungkin bisa dimainkan oleh hanya satu pemain.
+
+[Tingkat kesulitan](/wiki/id/Beatmap/Difficulty) TAG biasanya dilambangkan dengan penggunaan istilah "TAG" dalam nama kesulitan beatmap. Pada nama kesulitan "TAG" biasanya diikuti dengan angka, dengan khusus menunjukkan jumlah pemain yang harus bergabung untuk beatmap tersebut (misalnya, "TAG2" untuk dua pemain atau "TAG4" untuk empat pemain).
+
+Karena sifatnya, tingkat kesulitan TAG tidak diranked atau Loved, karena tingkat kesulitan TAG disengaja dirancang agar hampir mustahil dimainkan oleh hanya satu pemain.
```
Comment (mousesalt) [inline wiki/Beatmap/TAG/id.md:16]:
```suggestion
Karena sifatnya, tingkat kesulitan TAG tidak memiliki status Ranked maupun Loved, karena tingkat kesulitan TAG sengaja dirancang agar hampir mustahil dimainkan oleh satu pemain.
```

**Review by ZeroPyrozen — APPROVED** [review]
All good 👍

**Review by wowcake — APPROVED** [review]
looks good to me 👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**